src/main/java/com/customerApi/api/modals/Customer.java

```diff
@@ -0,0 +1,118 @@
+package com.customerApi.api.modals;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name="customer")
+public class Customer {
```

Comment:
Please format your class, coding is an art, code will also look beautiful if you format it properly.